src/App.vue

```diff
@@ -21,14 +21,38 @@ import FullWidth from '@/components/layouts/FullWidth.vue'
 import Navbar from '@/components/Navbar.vue'
 import MobileNavbar from '@/components/MobileNavbar.vue'
 
-import { between, required, email } from 'vee-validate/dist/rules'
+import { between, required, email, dimensions, image } from 'vee-validate/dist/rules'
 import { extend } from 'vee-validate'
 
+var validateImageDimensions = function (value, { width, height }) {
+  return new Promise(function (resolve) {
+    var image = new Image()
+    image.onerror = function () { return resolve(false) }
+    image.onload = function () { return resolve(image.width >= width && image.height >= height) }
+    image.src = URL.createObjectURL(value[0])
+  })
+}
+
+var validateImage = function (files) {
+  var regex = /\.(heic|heif|tiff|webp|ico|jpg|svg|jpeg|png|bmp|gif)$/i
```

Comment:
How are you going to display image formats like tiff?

https://stackoverflow.com/questions/2176991/display-tiff-image-in-all-web-browser

Reply:
It's just for sending image to server, where i will convert it to some other format (Preferably webp) and if it's svg it won't be converted.